sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/DefaultInputModule.java

```diff
@@ -27,11 +28,21 @@
 public class DefaultInputModule extends DefaultInputComponent implements InputModule {
 
   private final String moduleKey;
+  private final ProjectDefinition definition;
 
   public DefaultInputModule(String moduleKey) {
-    this.moduleKey = moduleKey;
+    this(ProjectDefinition.create().setKey(moduleKey), TestInputFileBuilder.batchId++);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS881)